recipes/crowcpp-crow/all/patches/001-disable-examples.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f00e9a9..79f432d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -29,7 +29,7 @@ if (MSVC)
+ 	set(Boost_USE_STATIC_LIBS "On")
+ 	find_package( Boost 1.52 COMPONENTS system thread regex REQUIRED )
+ else()
+-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++14 -pedantic -Wextra")
++	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++11 -pedantic -Wextra")
```

Comment:
Why do you need to change the C++ version?

Reply:
I've seen that no C++14 constructs are used, but C++11 is required by some boost header used in the project. Since the conan build system is using gcc 4.9, I've pushed it to C++11 instead of 14

Reply:
We try to minimize the patches we apply, but this one follows the draft rules #3903 so okay!